modules/labs-react/expandable/lib/ExpandableAvatar.tsx

```diff
@@ -0,0 +1,23 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import {jsx} from '@emotion/react';
+
+import React from 'react';
+
+import {createComponent, ExtractProps, useIsRTL} from '@workday/canvas-kit-react/common';
+import {Avatar} from '@workday/canvas-kit-react/avatar';
+import {space} from '@workday/canvas-kit-react/tokens';
+
+export interface ExpandableAvatarProps
+  extends Omit<
+    ExtractProps<typeof Avatar, never>,
+    keyof React.ButtonHTMLAttributes<HTMLButtonElement>
```

Comment:
I had to omit `HTMLButtonElement` props so `<Avatar>` would accept a `HTMLDivElement` ref. Let me know if you have a better idea of what to do here!

Reply:
I ran into something similar, we need to update Avatar so that this doesn't happen, all good!